internal/graph/graph_test.go

```diff
@@ -128,6 +128,20 @@ func TestCtorConflicts(t *testing.T) {
 	require.Contains(t, err.Error(), "ctor: func() (*graph.Child1, *graph.Child1, error), object type: *graph.Child1: node already exist for the constructor")
 }
 
+func TestValidateReturnTypes(t *testing.T) {
```

Comment:
test should describe the behavior at hand, i.e. `TestConstructorOverrideReturnsError`